pkg/sql/alter_table.go

```diff
@@ -324,7 +330,8 @@ func (n *alterTableNode) startExec(params runParams) error {
 					"session variable experimental_enable_primary_key_changes is set to false, cannot perform primary key change")
 			}
 
-			// Increment telemetry about uses of primary key changes.
+			// Increment telemetry about supported uses of primary key changes
```

Comment:
I feel like you should just delete this counter in favor of the new one you added

Reply:
sure thing!